provider/pkg/provider/environment_version_tags.go

```diff
@@ -163,8 +164,8 @@ func (evt *PulumiServiceEnvironmentVersionTagResource) Read(req *pulumirpc.ReadR
 	}
 
 	tag, err := evt.client.GetEnvironmentRevisionTag(ctx, input.Organization, input.Environment, input.TagName)
-	if err != nil {
-		return nil, fmt.Errorf("failed to read StackTag (%q): %w", req.Id, err)
+	if err != nil && !strings.Contains(err.Error(), "404") {
```

Comment:
Just checked, and unfortunately this does not work, because this method is trying to cast into `errorResponse`, but the root of the error returned by the client is just a string created with `fmt.Errorf`

I can create a helper method to make this check a bit more elegant, but unfortunately don't see a way around checking string for 404

Reply:
Bummer, thanks for trying it out